src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetAccountPAYESchemesForAuthorisedUser/WhenIGetAccountPayeSchemes.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Data;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.Levy;
+using SFA.DAS.EmployerAccounts.Models.PAYE;
+using SFA.DAS.EmployerAccounts.Queries.GetAccountPayeSchemes;
+using SFA.DAS.HashingService;
+using SFA.DAS.Validation;
+
+namespace SFA.DAS.EmployerAccounts.UnitTests.Queries.GetAccountPAYESchemesForAuthorisedUser
+{
+    class WhenIGetAccountPayeSchemes : QueryBaseTest<GetAccountPayeSchemesForAuthorisedUserQueryHandler, GetAccountPayeSchemesForAuthorisedUserQuery, GetAccountPayeSchemesResponse>
+    {
+        private const long AccountId = 2;
+        private static readonly DateTime UpdateDate = DateTime.Now;
+
+        private PayeView _payeView;
+        private DasEnglishFraction _englishFraction;
+        private Mock<IPayeSchemesService> _payeSchemesService;
+        private string _hashedAccountId;
+
+        public override GetAccountPayeSchemesForAuthorisedUserQuery Query { get; set; }
+        public override GetAccountPayeSchemesForAuthorisedUserQueryHandler RequestHandler { get; set; }
+        public override Mock<IValidator<GetAccountPayeSchemesForAuthorisedUserQuery>> RequestValidator { get; set; }
+       
+
+        [SetUp]
+        public void Arrange()
+        {
+            SetUp();
+
+            _payeView = new PayeView
+            {
+                AccountId = AccountId,
+                Ref = "123/ABC"
+            };
+
+            _englishFraction = new DasEnglishFraction
+            {
+                EmpRef = _payeView.Ref,
+                DateCalculated = UpdateDate,
+                Amount = 0.5m
+            };
+
+            _payeView.EnglishFraction = _englishFraction;
```

Comment:
_englishFraction isn't needed in the test any more as you're just doing an object comparison with whatever has been returned from the new service

Reply:
I just left it as it was part of the object being compared, but not necessary agreed. Removed.